### ISSUE TITLE
Can't stop TF from printing probabilities 

### ISSUE THREAD

**Issue by Jont828** [tensorflow/tensorflow#17088]
Please go to Stack Overflow for help and support:

https://stackoverflow.com/questions/tagged/tensorflow

If you open a GitHub issue, here is our policy:

1. It must be a bug, a feature request, or a significant problem with documentation (for small docs fixes please send a PR instead).
2. The form below must be filled out.
3. It shouldn't be a TensorBoard issue. Those go [here](https://github.com/tensorflow/tensorboard/issues).

**Here's why we have that policy**: TensorFlow developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

------------------------

### System information
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: Using the tutorial code
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: Arch Linux
- **TensorFlow installed from (source or binary)**: Binary
- **TensorFlow version (use command below)**: 1.5.0
- **Python version**:  3.6.4
- **CUDA/cuDNN version**:
- **GPU model and memory**:
- **Exact command to reproduce**: python cnn_mnist.py

You can collect some of this information using our environment capture script:

https://github.com/tensorflow/tensorflow/tree/master/tools/tf_env_collect.sh

You can obtain the TensorFlow version with

python -c "import tensorflow as tf; print(tf.GIT_VERSION, tf.VERSION)"

### Describe the problem

I can't stop TF from printing the "INFO:tensorflow:probabilities = []" part, where in the angle brackets is a list that spans for several hundred lines. Trying to suppress the verbosity either does nothing or causes lines like "INFO:tensorflow:loss = 2.314889, step = 2" to stop appearing as well. 

### Source code / logs

I used the source code from this page: https://www.tensorflow.org/tutorials/layers

**Comment by facaiy** [comment]
How about using tf.logging.set_verbosity(tf.logging.ERROR)?

**Comment by Jont828** [comment]
If I do that, it doesn't give me any output on the accuracy or results of the classifier.
